tools/test-coverage.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+CURRENT_GIT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+
+print_start_time() {
+    echo ""; echo "--- Started at $(date +"%d-%m-%Y %T")"; echo ""
+}
+
+print_finish_time() {
+    echo ""; echo "--- Finished at $(date +"%d-%m-%Y %T")"; echo ""
+}
+
+TEST_REPORT_FILE="."
+delete_the_old_test_report() {
+    TEST_REPORTS_FOLDER=".test-run-reports"
+    mkdir -p "${TEST_REPORTS_FOLDER}"
+    TEST_REPORT_FILE="${TEST_REPORTS_FOLDER}/test-report-${CURRENT_GIT_BRANCH}.html"
+    rm -f ${TEST_REPORT_FILE} && echo "Deleting old test report file: ${TEST_REPORT_FILE}"
+}
+
+COVERAGE_REPORT_FOLDER=".coverage-reports"
+COVERAGE_REPORT_FILE=""
+delete_the_old_coverage_report_folder_and_create_a_new_one() {
+    mkdir -p "${COVERAGE_REPORT_FOLDER}"
+    COVERAGE_REPORT_FOLDER="${COVERAGE_REPORT_FOLDER}/coverage-report-${CURRENT_GIT_BRANCH}"
+    rm -fr ${COVERAGE_REPORT_FOLDER} && echo "Deleting old test coverage folder: ${COVERAGE_REPORT_FOLDER}"
+    COVERAGE_REPORT_FILE="${COVERAGE_REPORT_FOLDER}/index.html"
+}
+
+test_run_exit_code=0
+SOURCES_FOLDER=kedro
+TESTS_FOLDER=tests
+TARGET_TEST_FOLDERS="$@"
+TARGET_TEST_FOLDERS="${TARGET_TEST_FOLDERS:-${TESTS_FOLDER}}"
+run_test_runner() {
+    echo ""; echo "~~~ Running tests with coverage on branch '${CURRENT_GIT_BRANCH}'"
+    set -x
+    pytest --cov-config default_coverage_report.toml     \
```

Comment:
Shouldn't it point to `pyproject.toml`?

Reply:
Yes I will have this changed, it was on my mind, and slipped away.